conf/gobblin-env.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Set Gobblin specific environment variables here.
+
+# Gobblin work dir
+export GOBBLIN_WORK_DIR=hdfs://localhost:8020/gobblin
+
+# Gobblin job configuration files
+export GOBBLIN_JOB_CONFIG_DIR=$GOBBLIN_WORK_DIR/jobs
```

Comment:
@lbendig, to address your second point. We have discussed changing GOBBLIN_WORK_DIR from an environment variable, to a command line config, and I do have some more thoughts on this I will document in a JIRA.

In terms of this pull request, I suggest we just address the issues @liyinan926 brought up above, remove ./gobblin-env.sh from gobblin-standalone.sh, and discuss any remaining feature requests in a separate JIRA